includes/Core/Modules/Modules.php

```diff
@@ -247,36 +247,6 @@ function( Module $module ) {
 			}
 		);
 
-		add_action(
```

Comment:
As mentioned in previous [comment](https://github.com/google/site-kit-wp/pull/8234#discussion_r1494528591), the remove analytics configuration was UA only, as it appears the whole block is. Otherwise, GA4 module is active always, as soon as Site kit is connected (without choosing to connect to Analytics as well)